src/main/java/com/archos/mediacenter/video/player/PlayerController.java

```diff
@@ -1265,7 +1265,7 @@ private int setProgress() {
                 if(mProgress2!=null)
                     mProgress2.setProgress((int) pos);
                 currentText = stringForTime(position);
-                endText = stringForTime(duration-position > 0 ? duration-position : 0);
+                endText = "-" + stringForTime(duration-position > 0 ? duration-position : 0);
```

Comment:
I prefer preserving the current view since it is explicit enough to have the remaining time on the right and there is no such thing that negative time...